### ISSUE TITLE
If `infotags` is blank then the recceiver crashes

### ISSUE THREAD

**Issue by simon-ess** [ChannelFinder/recsync#71]
If you have a configuration that has the following:
```
[cf]
infotags =
```
then the recceiver crashes on loading:
```
Traceback (most recent call last):
  File "/home/recceiver/venv/bin/twistd", line 8, in <module>
    sys.exit(run())
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/scripts/twistd.py", line 35, in run
    app.run(runApp, ServerOptions)
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/application/app.py", line 674, in run
    runApp(config)
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/scripts/twistd.py", line 29, in runApp
    runner.run()
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/application/app.py", line 374, in run
    self.postApplication()
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/scripts/_twistd_unix.py", line 254, in postApplication
    self.startApplication(self.application)
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/scripts/_twistd_unix.py", line 455, in startApplication
    app.startApplication(application, not self.config["no_save"])
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/application/app.py", line 689, in startApplication
    service.IService(application).startService()
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/application/service.py", line 276, in startService
    service.startService()
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/application/service.py", line 276, in startService
    service.startService()
  File "/home/recceiver/venv/lib/python3.10/site-packages/twisted/application/service.py", line 276, in startService
    service.startService()
  File "/home/recceiver/venv/lib/python3.10/site-packages/recceiver/cfstore.py", line 57, in startService
    self._startServiceWithLock()
  File "/home/recceiver/venv/lib/python3.10/site-packages/recceiver/cfstore.py", line 95, in _startServiceWithLock
    whitelist.append('recordDesc')
AttributeError: 'str' object has no attribute 'append'
```
This is due to
```py
wl = self.conf.get('infotags', list())
whitelist = [s.strip(', ') for s in wl.split()] \
    if wl else wl
```
if the infotags is empty (but present) then `wl` is an empty string, so we choose the _second_ branch of the conditional and return `whitelist` as an empty string.

**Comment by tynanford** [comment]
Hi @simon-ess , this should be fixed via #72

**Comment by simon-ess** [comment]
Thanks for the extremely quick response!
